src/utils.spec.js

```diff
@@ -0,0 +1,58 @@
+jest.mock('./queries.js')
+const { findSavedTripByDates } = require('./queries')
+const { keepOnlyNewTrips } = require('./utils')
+
+describe('remove duplicates', () => {
+  const trips = [
+    {
+      properties: {
+        start_fmt_time: '2021-01-01T12:00:00',
+        end_fmt_time: '2021-01-01T13:00:00'
+      }
+    },
+    {
+      properties: {
+        start_fmt_time: '2021-01-02T12:00:00',
+        end_fmt_time: '2021-01-02T13:00:00'
+      }
+    }
+  ]
+
+  it('should not include already existing trips', async () => {
```

Comment:
nit: I think we should have one test per scenario here, it will be easier to target the error when executing tests in the future. Or maybe at least we should add a message into each `expect()` method to discriminate it in the console output.

Reply:
Agreed :+1: